teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -130,6 +130,16 @@ public class ValidatorOptions {
       fallbackValue = "true")
   private boolean blockV3Enabled = ValidatorConfig.DEFAULT_BLOCK_V3_ENABLED;
 
+  @Option(
+      names = {"--exit-when-no-validator-keys-enabled"},
+      paramLabel = "<BOOLEAN>",
+      description = "Enable allowing no loaded validator keys",
```

Comment:
we'll need to do some more work on this description

Reply:
Just fixing it now